storybook/src/docs/components/DataGrid/DataGrid.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import { gql, useQuery } from "@apollo/client";
 import {
     CrudContextMenu,
+    GridColDef as CometGridColDef,
```

Comment:
That's exactly why I'd use an import restriction 😁 Having to rename imports sucks